tests/models/test_category.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import json
+
+from flask_sqlalchemy import SQLAlchemy
+from app import app
+from models import Category
+
+class CategoryTestCase(unittest.TestCase):
+  def setUp(self):
+    self.client = app.test_client()
+    self.db = SQLAlchemy(app)
```

Comment:
This might not even be worth the time it'll take to fix, commit, and push, but I think we can get a tiny performance boost by importing db from app
```suggestion
from app import app, db
from models import Category

class CategoryTestCase(unittest.TestCase):
  def setUp(self):
    self.client = app.test_client()
    self.db = db
```
```
➜  freefrom-map-backend git:(category-model) python3 -m unittest
./Users/kimd/.pyenv/versions/3.8-dev/lib/python3.8/unittest/case.py:1215: DeprecationWarning: assertDictContainsSubset is deprecated
  warnings.warn('assertDictContainsSubset is deprecated',
..
----------------------------------------------------------------------
Ran 3 tests in 0.146s

OK
➜  freefrom-map-backend git:(category-model) vim tests/models/test_category.py
➜  freefrom-map-backend git:(category-model) ✗ python3 -m unittest              
./Users/kimd/.pyenv/versions/3.8-dev/lib/python3.8/unittest/case.py:1215: DeprecationWarning: assertDictContainsSubset is deprecated
  warnings.warn('assertDictContainsSubset is deprecated',
..
----------------------------------------------------------------------
Ran 3 tests in 0.121s

OK
```

Reply:
Good call! Thank you :)